pom.xml

```diff
@@ -98,23 +99,23 @@
             <groupId>nl.dtl</groupId>
             <artifactId>fairmetadata4j</artifactId>
             <version>${fairmetadata.version}</version>
-        </dependency>
+        </dependency>        
         <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-api</artifactId>
-            <version>${log4j.version}</version>
-        </dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>${slf4jVersion}</version>
+        </dependency>   
         <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-core</artifactId>
-            <version>${log4j.version}</version>
-        </dependency>    
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+            <version>${slf4jVersion}</version>

```

Comment:
Any reason for log4j version 1.x instead of 2.x? We can use log4j-slf4j-impl the dependency for log4j 2.x.